code-style/git/README.md

```diff
@@ -322,8 +335,11 @@ Voila! You're done. You've successfully rebased a branch onto the master branch!
 [Draft PRs]: https://github.blog/2019-02-14-introducing-draft-pull-requests/
 [pull_request]: https://help.github.com/articles/using-pull-requests
 [Sparkbox]: http://seesparkbox.com
-[Sprintly]: https://sprint.ly/
-[pull request template]: https://seesparkbox.com/foundry/better_pull_requests_merge_requests_with_templates
+[pull_request_template]: https://seesparkbox.com/foundry/better_pull_requests_merge_requests_with_templates
 [example PR template]: ./PULL_REQUEST_TEMPLATE.md
 [example issue template]: ./ISSUE_TEMPLATE.md
 [Create a branch]: #naming-branches
+[curate your commit history]: #curating-your-commit-history
+[Conventional Commits]: https://www.conventionalcommits.org/en/v1.0.0/
+[Taking Control of your Commit History]: https://seesparkbox.com/foundry/take_control_of_your_commit_history
+[onboard the reviewer to your pull request]: https://sparkbox.com/foundry/interactive_rebasing_curates_commits_to_speed_up_pull_request_review_process
```

Comment:
Very minor suggestion: Using reference links without titles (what you're doing here) is great if that link is used only once. But as soon as you have _multiple_ texts that are using it, it might make more sense to assign titles per instance. The reason why is because, in order for these links to work, you're saying that the title for each of these links ("onboard the reviewer to your pull request") **must** match. But this is easily broken in the event that the phrasing for one of these sentences changes.

If it was only being used once, that'd be easy to track (i.e., someone would see that the title is changing so they'll have to change the reference link). But if it's being used twice, people might not realize they'd actually have to change the title twice.

Instead, I'd recommend doing something like this:

```suggestion
[foundry_curated_commits]: https://sparkbox.com/foundry/interactive_rebasing_curates_commits_to_speed_up_pull_request_review_process
```

And then update the text where it's being used.

```

[onboard the reviewer to your pull request][foundry_curated_commits]